packages/corelib/src/dataModel/Rundown.ts

```diff
@@ -21,14 +20,20 @@ export interface RundownImportVersions {
 }
 
 /** This is a very uncomplete mock-up of the Rundown object */
-export interface Rundown
-	extends ProtectedStringProperties<IBlueprintRundownDB, '_id' | 'playlistId' | 'showStyleVariantId'> {
+export interface Rundown {
```

Comment:
I dont think so. As long as we can still convert between them fairly easily I see no harm.
We are already using a method to manually convert a DBRundown to IBlueprintRundownDB, so we dont need the types to 100% line up anyway.

I would argue that we should do this separation to all of the types we give to the blueprints. It avoids some of the tedious typescript hacks which arent all that type safe (eg `ProtectedStringProperties`), and will let us structure the blueprint interfaces in a way that makes sense to them, rather than following the mongodb data structure.
It will also help simplify out typings, as Rundown will be defined in one place, rather than being split across two separate packages. It can be really hard to figure out what properties Rundown has, with some being on Rundown, some being from IBlueprintRundown, and some members of IBlueprintRundown being Omitted and overwritten

Reply:
I'm not really against the idea, but I don't think it should be included in _this_ PR.

(it sounds like something we could talk about on the next dev meet tho)

Reply:
Yeah thats reasonable.

At one point in this branches life it had more motivation as _rank was wanted on IBlueprintRundown but not Rundown, but as I removed _rank from both there doesnt appear to be a need to separate them anymore